### ISSUE TITLE
Path with double context in windows for --add option

### ISSUE THREAD

**Issue by o-Chamie-o** [pixari/vue-i18n-extract#98]
I think I found a bug.

When I launch : 
```
npx vue-i18n-extract report -v "./src/**/*.?(vue)" -l "./src/locales/**/*.json" -a
```

I got this result : 

```(node:6724) UnhandledPromiseRejectionWarning: Error: ENOENT: no such file or dir
ectory, open 'C:\repoC:\repo\src\locales\en.json'
    at Object.openSync (fs.js:462:3)
    at Object.writeFileSync (fs.js:1384:35)
    at C:\laragon\www\oklinda\node_modules\vue-i18n-extract\dist\vue-i18n-extrac
t.umd.js:189:12
```

Here is the fix I did : 
In `vue-i18n-extract.umd.js` line 185
```
function writeMissingToLanguage(resolvedLanguageFiles, missingKeys) {
    const languageFiles = readLangFiles(resolvedLanguageFiles);
   [...]
      const fileExtension = languageFile.fileName.substring(languageFile.fileName.lastIndexOf('.') + 1);
      const filePath = path.resolve(process.cwd() + languageFile.fileName);
      const stringifiedContent = JSON.stringify(languageFileContent, null, 2);

     [...}
    });
  }
```

Remove the `process.cwd()` as the path are already in the context.

```
function writeMissingToLanguage(resolvedLanguageFiles, missingKeys) {
    const languageFiles = readLangFiles(resolvedLanguageFiles);
   [...]
      const fileExtension = languageFile.fileName.substring(languageFile.fileName.lastIndexOf('.') + 1);
      const filePath = path.resolve(languageFile.fileName);
      const stringifiedContent = JSON.stringify(languageFileContent, null, 2);

     [...}
    });
  }
```

Now it works for me.

**Comment by esfraner** [comment]
I had the same problem, I did what you said in your comment and it worked perfectly.  
@o-Chamie-o thanks!

**Comment by esfraner** [comment]
The problem only appears if you run it on Windows, it works perfectly on Linux.

**Comment by esfraner** [comment]
> > The problem only appears if you run it on Windows, it works perfectly on Linux.
> 
> Hum interesting, clearly the `process.cwd()` don't have the same behavior in Windows and in Linux then.
> 
> Can you try my fix in linux too, just to see if it broke it or not?
> 
> Thanks for the update. :)

If I try to do it in linux I find the following error:
![image](https://user-images.githubusercontent.com/22304628/105344980-f91c0900-5be3-11eb-8dbf-7354bcc401d3.png)

**Comment by o-Chamie-o** [comment]
So maybe this is more a windows bug then. :thinking:

**Comment by gilles-crealp** [comment]
It's a typo; you must use `,` with `path.resolve` not `+`...

It has been solved in #94, update to version 1.1.11.

**Comment by o-Chamie-o** [comment]
> It's a typo; you must use `,` with `path.resolve` not `+`...
> 
> It has been solved in #94, update to version 1.1.11.

Oh how come I didn't see it. :o 

Yeah it's a typo, just try you fix.

Thanks a bunch for the update @gilles-crealp. :)
